### DISCUSSION TITLE
Duda respecto a informe

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#6] [Tarea 0] [answered]
Tengo dos dudas respecto a la realización del informe, primero que todo la segunda pregunta del informe, la idea es resolverlo con pseudocódigo o con palabras no más?
Además, asumo que las referencias/bibliografía no es considerada en el largo de dos planas, pero quería asegurarme de esto.

**Comment by sachondo7** [comment, marked as answer]
El formato es libre, si quieres puedes usar pseudocodigo, palabras, algún diagrama o lo que te permita a ti explicar efectivamente como implementar esos comandos según el modelamiento que propones. 

Con respecto a las referencias esta bien lo que dices, no cuenta en el largo de dos planas.
